extension/modules/personas.js

```diff
@@ -1123,8 +1123,8 @@ var PersonaController = {
         // Update the Custom menu. Custom personas unavailable in per-window
         // personas mode.
         let customMenu = document.getElementById("personas-plus-custom-menu");
-        customMenu.hidden = this._prefs.get("perwindow") || !this._prefs.get("showCustomMenu");
-        if (!customMenu.hidden) {
+        customMenu.disabled = this._prefs.get("perwindow")
```

Comment:
Hm, works for me.

<img width="230" alt="screen shot 2016-07-16 at 22 53 05" src="https://cloud.githubusercontent.com/assets/873258/16897119/1ebe3260-4ba8-11e6-8669-14ca434d511a.png">